mithril-common/src/crypto_helper/cardano/opcert.rs

```diff
@@ -57,9 +60,9 @@ impl OpCert {
         kes_vk: KesPublicKey,
         issue_number: u64,
         start_kes_period: u64,
-        cold_keypair: EdKeypair,
+        cold_keypair: EdSecretKey,
```

Comment:
if you choose to go with this approach , the name of this should be `cold_secret_key`